mmaction/datasets/activitynet_dataset.py

```diff
@@ -166,6 +175,48 @@ def dump_results(self, results, out, output_format, version='VERSION 1.3'):
                 'external_data': {}
             }
             mmcv.dump(output_dict, out)
+
+            if classifier_input:
+                with open(out) as f_out:
+                    proposal_results = json.load(f_out)['results']
+                videos = proposal_results.keys()
+                video_idx = 0
+
+                out_classifier_input = open(
+                    self.ann_file.replace(
+                        osp.split(self.ann_file)[1],
+                        'anet_val_classifier_input.txt'), 'w')
```

Comment:
Is SSN using this hardcoded filename as default?

Reply:
No, now SSN only supports THUMOS14.